src/playwright.ts

```diff
@@ -0,0 +1,106 @@
+const testsGenerator: Fig.Generator = {
+  custom: async (tokens, executeShellCommand) => {
+    // TODO: load the list of test files specified in testDir on playwright.config file
+    return [] as Fig.Suggestion[];
+  },
+};
+
+const browserSuggestions: Fig.Arg["suggestions"] = [
+  {
+    name: "chromium",
+    displayName: "Chromium",
+  },
+  {
+    name: "chrome",
+    displayName: "Chrome",
+  },
+  {
+    name: "chrome-beta",
+    displayName: "Chrome Beta",
+  },
+  {
+    name: "msedge",
+    displayName: "Microsoft Edge",
+  },
+  {
+    name: "msedge-beta",
+    displayName: "Microsoft Edge Beta",
+  },
+  {
+    name: "msedge-dev",
+    displayName: "Microsoft Edge Dev",
+  },
+  {
+    name: "firefox",
+    displayName: "Firefox",
+  },
+  {
+    name: "webkit",
+    displayName: "WebKit",
+  },
+];
+
+const helpOption: Fig.Option = {
+  name: ["--help", "-h"],
+  description: "Display help for command",
+  priority: -1,
```

Comment:
-1 is not a valid priority, to always have this at the bottom of the list use a priority : 0 <= p <= 25
You can also add `isPersistent: true` to the --help option so that each subcommand of the spec will have it.
```suggestion
  priority: 1,
```